include/megaapi.h

```diff
@@ -477,6 +478,21 @@ class MegaNode
          */
         virtual const char* getFingerprint();
 
+        /**
+         * @brief Returns the original fingerprint (Base64-encoded) of the node
+         *
+         * In the case where a file was modified before uploaded (eg. resized photo or gps coords removed),
+         * it may have an original fingerprint set (by setOriginalFingerprint() or backgroundMediaUploadComplete()),
```

Comment:
In MegaApi's documentation, methods are written without the parenthesis usually. Something like `MegaApi::setOriginalFingerprint`. I'll fix it